#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
uint8 pitch_rate
uint8 lat_acceleration
